src/adapters/input-adapter.ts

```diff
@@ -125,7 +126,7 @@ export class InputAdapter implements AdapterInterface {
     assertElementIsDisplayed(this.element);
     const alignedMatches = lookupMatches(this.lastContentChecked!, this.getCurrentText(), matches, 'TEXT');
 
-    if (_.isEmpty(alignedMatches)) {
+    if (alignedMatches.length === 0) {
```

Comment:
The lodash `_.isEmpty(null)` would return true ([docs](https://lodash.com/docs#isEmpty)). So I would suggest to replace it with `if(!alignedMatches || alignedMatches.length === 0)`

Reply:
fixed